### PR TITLE
[READY]Lavaland elites rework

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

The point of lavaland elites was the tumor boosting part, and also getting killed by sentient almost-megafauna is cool. Removing it isn't cool. So, this PR returns tumor boosting, but also fixes some problems that they had.

So, the first part - boosting the tumor removes pandora from possible spawns. Pandora is cringe, AOE blasts that you almost can't dodge aren't fun.

Second part: Now shitty kinetic crusher trophy's aren't the main reward for killing them. They now will be get by killing elites with your KA. Instead, boosted elites drop special loot:

Prophet's guard - Herald - A kinetic crusher trophy, you gain it by killing Herald with a KC. Every mark detonation it has a chance to spawn a mirror with limited lifespawn, that will shoot projectiles when you will detonate a mark near it.

Goliath tentacle - Broodmother - a armblade-like weapon. Deals 15 brute damage, double damage to mobs and triple damage to megafauna/elites. Clicking a title while on disarm intent will spawn a tentacle there, consuming a charge. Charges regenerate after time.

Lesser necromancy wand - Legionare - Allows to summon a player controlled skeleton simplemob to support you. Can sustain only one at time.

Pandora's wraith - Pandora - A KC mod, you get it by killing it with a KC. On mark detonation has a chance to summon 3x3 hierophant blasts.

Also funny shard will have much less chance to drop.

Third part: 

Elites will be able to alt+click a mining shuttle console to call a shuttle, and when they won there will be an announcement, so people can prepare for the fight. And now they are large mobs, meaning that they can be marked with a KC.

Also they will emit a GPS signal, E.

# Wiki Documentation

Add loot information

# Changelog

Hope it will not be closed just because it contains a revert as a part of it

:cl:  
rscadd: Returns tumor boosting
rscadd: New elite loot
rscadd: no more sentient pandoras
rscadd: Elites can now call the mining shuttle
rscadd: Elites now emit a GPS signal
imageadd: Sprites for new elite loot
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
I have realised that herald already has a normal loot item, i will maybe make elites have not only one possible drop item

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm [hunk 1]
```diff
@@ -147,19 +148,41 @@ While using this makes the system rely on OnFire, it still gives options for tim
...
+					elitemind.playsound_local(get_turf(elitemind), 'sound/effects/magic.ogg', 40, 0)
+					to_chat(elitemind, "<b>You have been chosen to play as a Lavaland Elite.\nIn a few seconds, you will be summoned on Lavaland as a monster to fight your activator, in a fight to the death.\nYour attacks can be switched using the buttons on the top left of the HUD, and used by clicking on targets or tiles similar to a gun.\nWhile the opponent might have an upper hand with  powerful mining equipment and tools, you have great power normally limited by AI mobs.\nIf you want to win, you'll have to use your powers in creative ways to ensure the kill.  It's suggested you try using them all as soon as possible.\nShould you win, you'll receive extra information regarding what to do after.  Good luck!</b>")
+					addtimer(CALLBACK(src, .proc/spawn_elite, elitemind), 100)
+				else
+					visible_message(span_boldwarning("The stirring stops, and nothing emerges.  Perhaps try again later."))
+					activity = TUMOR_INACTIVE
+					activator = null
+				
 
 obj/structure/elite_tumor/proc/spawn_elite()
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]:
```suggestion
obj/structure/elite_tumor/proc/spawn_elite(var/mob/dead/observer/elitemind)
```

**Comment by Marmio64** [comment]
Pandora was like the one challenging player controlled elite to fight and was rather fun to play as.

**Comment by ghost** [comment]
> Pandora was like the one challenging player controlled elite to fight and was rather fun to play as. 

I agree, but it is really to OP. It is like the strongest lavaland elite, because of large aoe blasts that is really hard to dodge, and making such mob player controlled leads to crew getting crushed with no chances to survive.

**Comment by ghost** [comment]
Also fuck mirror shards

**Comment by ghost** [comment]
Seems partly finished, i only need to add icons and i don't know will the console altclick part work

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm [hunk 2]
```diff
@@ -245,6 +306,18 @@ obj/structure/elite_tumor/proc/onEliteWon()
 	activity = TUMOR_PASSIVE
 	activator = null
 	mychild.revive(full_heal = TRUE, admin_revive = TRUE)
+	if(boosted)
+		times_won++
+		mychild.maxHealth = mychild.maxHealth * 0.5
+		mychild.health = mychild.maxHealth
+		if(!mychild.elitegps)
+			addtimer(CALLBACK(src, .proc/add_gps), 2 MINUTES)
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm]:
```suggestion
			addtimer(CALLBACK(mychild, .proc/add_gps), 2 MINUTES)
```

**Comment by ghost** [comment]
Are ya fucking demented or what

**Comment by ghost** [comment]
Fucking merge conflicts

**Comment by ghost** [comment]
Yeeeeee

**Comment by ghost** [comment]
FUCK

**Comment by ghost** [comment]
Okay

**Comment by ghost** [comment]
Finaly

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/herald.dm [hunk 3]
```diff
@@ -266,9 +269,82 @@
...
+
+
+
+//KC loot
+
+/obj/item/crusher_trophy/prophets_guard
+	name = "prophet's guard"
+	desc = "A trophy, that attracts unknown powers to guard you."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "prophets_guard"
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/herald.dm]:
```suggestion
	icon_state = "prophets_talisman"
```

**Comment by ghost** [comment]
Ready 100%

**Comment by SomeguyManperson** [comment]
Miners shouldn't be turning people into murderbone ghostroles by failing a boss fight and elites cause this which is one among many reasons I had the sentience effect removed and no I have not changed my mind about it

**Comment by ghost** [comment]
Bruh

**Pull request closed without merging**